sgkit/tests/io/vcf/test_vcf_reader.py

```diff
@@ -64,7 +64,7 @@ def test_vcf_to_zarr__small_vcf(shared_datadir, is_path, tmp_path):
             [[0, 0], [0, 0], [0, 0]],
             [[0, 1], [0, 2], [-1, -1]],
             [[0, 0], [0, 0], [-1, -1]],
-            [[0, -2], [0, 1], [0, 2]],  # -2 signifies non-allele
+            [[0, -1], [0, 1], [0, 2]],
```

Comment:
Presumably because this is tested more extensively below now?

Reply:
Yes, I wanted to add a more extensive test for the mixed ploidy and polyploid (tetraploid) cases so I left this case as fixed ploidy. This resulted in the reversion of -2 to -1 due to this [change](https://github.com/pystatgen/sgkit/pull/373/files#diff-0eaa71fa8d1bd0b47476d65927c49ac5e40a1856df5d4c56719fcbced1a2766dR110) defaulting to -1 for the fill value.